.github/workflows/publish-docker.yml

```diff
@@ -79,6 +83,7 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
+          file: ${{ inputs.dockerFile }}
```

Comment:
I am not sure if I need to provide a fallback here too, or if this will automatically resolve to {context}/Dockerfile by docker/build-push-action@v3.

Reply:
you can set up a test PR in e.g. PVS pointing to this branch and make sure it still works, as it runs on PRs as well

Reply:
Looks good to me: https://github.com/ftrackhq/pvs/actions/runs/3530070991/jobs/5921672650

I'll keep this PR open until I have verified that ftrackhq/frontend#238 passes as well.